region/brinstar/blue/Blue Brinstar Energy Tank Room.json

```diff
@@ -841,6 +841,7 @@
       "name": "Artificial Morph Remotely Collect the Item",
       "requires": [
         "canEnterGMode",
+        {"itemNotCollectedAtNode": 4},
```

Comment:
[6, 4] collects the item, so this wouldn't work.

Reply:
Putting the `collectsItem` where GMode ends is better, but remote collections have been written where you pick up the item and then leave to end gmode in the following strat.

Reply:
I want to go through all of the remote acquire again soonish, and use `collectsItem`. I bet there are some strats where it's expected to use the item obtained during the remote acquire, even though the item isn't usable yet. (Map Rando won't have this problem, but it still feels like an important fix I want to prioritize)